core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -491,17 +491,8 @@ private void resetByDuration(final Consumer<byte[], byte[]> client,
                                  final Duration duration) {
         final Instant now = Instant.now();
```

Comment:
Could we remove those local variables? For example:
```java

        resetToDatetime(client, inputTopicPartitions, Instant.now().minus(duration).toEpochMilli());
```

Reply:
Thank you for your information. I have updated this problem.